added template togepi

BenChenDev opened the pull request.

Commit:
added template togepi

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+    const Profile = () => (
+        <div className="flex justify-start">
+            <div className="bg-gray-400">
+                <img className="w-40 h-40" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Circle-icons-profile.svg/1200px-Circle-icons-profile.svg.png" alt="profile"/>
```
remove the profile image section. A profile image is not applicable in a Canadian resume

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+        data.education &&
+        config.education.enable && (
+            <div className="mb-6">
+                <Heading title="EDUCATION" />
```
do not use hardcoded string. see other templates for examples

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+        data.work &&
+        config.work.enable && (
+            <div>
+                <Heading title="WORK EXPERIENCE" />
```
do not use hardcoded string. see other templates for examples

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+        data.certifications &&
+        config.certifications.enable && (
+            <div>
+                <Heading title="CERTIFICATE" />
```
do not use hardcoded string. see other templates for examples

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+        data.skills &&
+        config.skills.enable && (
+            <div>
+                <Heading title="SPECIAL SKILLS"/>
```
do not use hardcoded string. see other templates for examples

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+        data.basics.summary &&
+        config.summary.enable && (
+            <div>
+                <Heading title="ABOUT ME" />
```
do not use hardcoded string. see other templates for examples

SHENG-X replied:
use Summary for title

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+            <div className="grid grid-cols-12 mb-5">
+                <div className="col-span-1 mt-8 h-8" style={{background: theme.colors.primary}}/>
+                <div className="col-start-2 col-span-11 mt-8 text-white h-8" style={{backgroundColor: theme.colors.accent}}>
+                    <h1 className="ml-3 mt-1">RESUME</h1>
```
do not use hardcoded string. see other templates for examples

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+                        />
+                        <ContactItem 
+                            icon="linkedin" 
+                            value={formatDisplayURL(data.basics.linkedin)}
```
you don't have to a put line break here

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+                </div>
+            </div>
+            
+            <div style={{'padding':'0 40px'}}>
```
padding at the bottom is needed. otherwise, the content of the about me(summary) will be cut out

SHENG-X replied:
use class px- pb- here instead of inline styling

SHENG-X commented on src/templates/togepi/index.js:
```diff
@@ -0,0 +1,5 @@
+import Togepi from './Togepi';
+import image from '../onyx/preview.png';//use preview of Onyx for testing
```
add todo here. ex: // TODO: replace with correct preview image

SHENG-X commented on src/templates/togepi/Togepi.js:
```diff
@@ -0,0 +1,262 @@
+    );
+
+    const EducationItem = x => (
+        <div key={x.id} className="">
```
remove className="" if no class name was there

SHENG-X reviewed (commented):
Look good. There are small issues need to be fixed

Commit:
resolved small issues

SHENG-X committed:
chore: togepi preview image

SHENG-X committed:
refactor: togepi template styles

- keeped line divider hight the same 
- introduced Title component that manages data, location, and name information
- converted skills section into 2 columns bullet point format
- minor fix on the margin and padding
